trade-finance-manager-ui/server/api.js

```diff
@@ -1230,6 +1230,21 @@ const addFeesToAnExistingPayment = async (reportId, feeRecordIds, paymentIds, us
   return response.data;
 };
 
+/**
+ * @param {string} dealId - The deal ID
+ * @param {Partial<import('@ukef/dtfs2-common').TfmDealCancellation>} cancellationUpdate - The deal cancellation update object
+ * @param {string} userToken - The user token
+ * @returns {Promise<void>}
+ */
+const updateDealCancellation = async (dealId, cancellationUpdate, userToken) => {
+  await axios({
```

Comment:
try-catch block please.

Reply:
My expectation is that this will be called by the controller that will wrap this in a `try/catch` block itself (see implementation of the e.g. `addFeesToAnExistingPayment`, `removeFeesFromPayment` `markKeyingDataAsDone` functions in this file. Adding a try/catch block here as well would be unnecessary and reduce DRY of the code

Reply:
I think this would only be beneficial, regardless of how it's called and if the caller is wrapped in a try/catch. Seems that half of the functions in this file have a try/catch, the other half don't. I think having a try/catch cannot hurt. Helpful for debugging

Reply:
@abhi-markan @ttbarnes I've added a try/catch block here as discussed on Friday 👍

Reply:
Thank you @BethThomas141